yarn.lock

```diff
@@ -502,10 +502,10 @@
   dependencies:
     "@types/react" "*"
 
-"@types/react-select@*", "@types/react-select@^1.2.3":
-  version "1.2.3"
-  resolved "https://registry.yarnpkg.com/@types/react-select/-/react-select-1.2.3.tgz#95b3af27e73d8fd4ba421c3dcfc2ce97f2603f92"
-  integrity sha512-D/A5W0b0SKxdFszwCp3AuVoMNS0+2w5YscFlUr8ue/Fycwm8mlS17H7MwMi37ByvjcgmCZeIK5FWSHIytHBfQg==
+"@types/react-select@^1", "@types/react-select@^1.2.4":
+  version "1.3.4"
```

Comment:
Thanks for updating this.  Was there a typing problem that this resolved?  If so, I'd prefer to commit this separately with a nice comment about what this fixes.

Reply:
Yes it was not recognizing the generics and causing several errors. I will update separately.